packages/admin-next/dashboard/src/components/layout/notifications/notifications.tsx

```diff
@@ -1,9 +1,35 @@
-import { BellAlert } from "@medusajs/icons"
-import { Drawer, Heading, IconButton } from "@medusajs/ui"
+import {
+  ArrowDownTray,
+  BellAlert,
+  InformationCircleSolid,
+} from "@medusajs/icons"
+import { Drawer, Heading, IconButton, Label, Text } from "@medusajs/ui"
 import { useEffect, useState } from "react"
+import { useTranslation } from "react-i18next"
+import { useNotifications } from "../../../hooks/api"
+import { HttpTypes } from "@medusajs/types"
+import { formatDistance } from "date-fns"
+import { Divider } from "../../common/divider"
+
+interface NotificationData {
+  title: string
+  description?: string
+  file?: {
+    filename?: string
+    url?: string
+    mimeType?: string
+  }
+}
 
 export const Notifications = () => {
   const [open, setOpen] = useState(false)
+  const { t } = useTranslation()
+
+  // TODO: Make it infinitely scrollable
+  const { notifications } = useNotifications(
```

Comment:
@kasperkristensen I see we have an infinite scroll for the combobox, is there also some other component/list that is infinitely scrollable already?

Reply:
We don't have a component for it, it's more a matter of setting up the `useInfiniteQuery` from `@tanstack/react-query`. You could properly re-use the `useComboboxData` hook, and just not make use of the search related values.